rules/valid-describe.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const describeAliases = Object.assign(Object.create(null), {
+  describe: true,
+  'describe.only': true,
+  'describe.skip': true,
+  fdescribe: true,
+  xdescribe: true,
+});
+
+const getNodeName = node => {
+  if (node.type === 'MemberExpression') {
+    return node.object.name + '.' + node.property.name;
+  }
+  return node.name;
+};
+
+const isDescribe = node =>
+  node.type === 'CallExpression' && describeAliases[getNodeName(node.callee)];
+
+const isFunction = node =>
+  node.type === 'FunctionExpression' || node.type === 'ArrowFunctionExpression';
+
+const isAsync = node => node.async;
+
+const hasParams = node => node.params.length > 0;
+
+module.exports = {
+  meta: {
+    docs: {
+      url:
+        'https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/valid-describe.md',
+    },
+  },
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (node && isDescribe(node)) {
```

Comment:
Can `node` ever be falsy here? I'm not sure I need to include this `node &&`.

Reply:
not sure. Seems weird if it can be?

Reply:
Does seem weird, will remove that check